setup.py

```diff
@@ -78,7 +78,7 @@
         "matplotlib         >= 3.3",
         "numba",
         "numpy              >= 1.17.3",
-        "orix               >= 0.9",
+        "orix               >= 0.11",
```

Comment:
Should this not be 0.12.1 now? Or is dependency updating perhaps done separately?

Reply:
Yes :)